scripts/cleanup_openstack_instance_snapshot.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""Cleanup unused instance snapshot from glance repository
+
+Usage: scripts/cleanup_openstack_instance_snapshot.py [optional list of provider keys]
+
+If no providers specified, it will cleanup all of them.
+
+"""
+import sys
+from datetime import datetime
```

Comment:
Is it okay to put it as 
`from datetime import datetime, timedelta` ?

Reply:
Imports should usually be on separate lines, but if we import 2 sub-module from same parent modules then we can right it on single line, but again i preferred to make it on separate line.
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
pre-commit configuration is using reorder-python-imports, which enforces one import per line.  Not my personal preference, but the standard we chose to adhere to since its enforced automatically.